packages/api/endpoints/distribution-metrics.js

```diff
@@ -0,0 +1,243 @@
+'use strict';
+
+const AWS = require('aws-sdk');
+const flatten = require('lodash.flatten');
+const router = require('express-promise-router')();
+
+const {
+  aws: { cloudwatch },
+  testUtils: { randomId }
+} = require('@cumulus/common');
+
+/**
+ * @param {Array<numbers>} array
+ * @returns {number} sum of values in array
+ */
+const sumArray = (array) => array.reduce((accum, cur) => accum + cur, 0);
+
+const callGetStages = (restApiId) => {
+  const apiGateway = new AWS.APIGateway();
+  return apiGateway.getStages({ restApiId }).promise();
+};
+
+/**
+ *
+ * @param {Object} getStagesResult - return value of aws.apigateway.getStages on a given apiId
+ * @returns {Array<string>} - list of stage names associated with the apiId
+ */
+const parseStagesResult = (getStagesResult) =>
+  getStagesResult.item.map((i) => i.stageName);
+
+/**
+ * Get a list of the all stages associated with the url's api Gateway.
+ *
+ * @param {string} restApiId - api endpoint where the first part of the host is the restApiId.
+ * @returns {Promise<Array>} - array of names of stages found on api gateway.
+ */
+const getApiStages = (restApiId) =>
+  callGetStages(restApiId).then(parseStagesResult);
+
+/**
+ * Lists all stages associated with the backend and distribution api urls.
+ */
+const listAllStages = async () => {
+  const apiId = process.env.distributionApiId;
+  return getApiStages(apiId);
+};
+
+/**
+ * Returns the name of the stage used in the cumulus deployment.
+ */
+const getStageName = async () => {
+  const stages = Array.from(new Set(await listAllStages()));
```

Comment:
I wish we had slack history because I talked about this with Patrick (and you?) My memory, however bad, is that originally I understood that we'd use the stages to version the API, eventually deploying to multiple stages for each version.  

Now that we use Express and route all of the requests, the versioning can be handled in software, and we shouldn't ever need multiple stages.   And Patrick said we needed to rename it from the current `dev` 

I'm not sure where even you would document that we only have one stage.  Ideas?.

Reply:
I would say in the deployment docs somewhere?

Reply:
Added a footnote.  It'll will probably get removed at somepoint I suspect.